docs/user-guide/retrieving_data.rst

```diff
@@ -0,0 +1,117 @@
+.. _retrieving_data:
+
+**************************************
+Retrieving and Downloading Data
+**************************************
+
+The `~swxsoc.util.util.SWXSOCClient` class in the `swxsoc` package is a FIDO client used for querying and downloading data from the SWxSOC buckets. Below are some examples demonstrating how to use this client for searching and downloading data.
+
+.. warning::
+
+    To download the data, you must have access to the SWxSOC buckets. If you do not have access, please contact the SWxSOC team. The client utilizes either presigned S3 URLs via the `boto3` package to download the data if you have AWS Credentials or it can download via S3 URLs if you have access to the SWxSOC buckets via allowed IP addresses.
+
+Attributes for Searching Data
+=============================
+The `~swxsoc.util.util.SWXSOCClient` class supports the following attributes for searching data:
+
+- `Instrument`: The instrument name for the data (e.g., 'EEA', 'SHARP').
+- `Level`: The data level (e.g., 'L1', 'L2').
+- `SearchTime`: The time range for the data (e.g., `('2021-01-01T00:00:00Z', '2021-01-02T00:00:00Z')`).
+- `DevelopmentBucket`: Whether to search in the development bucket (e.g., `True`, `False`).
+
+Examples for Searching Data
```

Comment:
Can you give an example of what the results should look like, even if they are hard-coded?